packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -278,6 +280,11 @@ class DayPicker extends StatelessWidget {
   /// Optional user supplied predicate function to customize selectable days.
   final SelectableDayPredicate selectableDayPredicate;
 
+  /// If set to [DragStartBehavior.start], picker drag behavior will begin upon the
```

Comment:
The doc comment should follow our style guide:

/// One sentence explanation.
///
/// More details...